www/docs/en/dev/guide/platforms/ios/webview.md

```diff
@@ -220,5 +222,16 @@ After using either of these two methods, continue with the **"Using CDVViewContr
     */
     viewController.wwwFolderName = @"myfolder";
     viewController.startPage = @"mypage.html"
-    ```
+    ```
+## Adding Plugins files used in you cordova project
```

Comment:
It's just that you skipped over all the parts of how they get the plugin source in the first place. That part is missing, since your steps assume a CLI workflow (assuming a Plugins folder and such).

In a CLI workflow, they just do "cordova plugin add". In an embedded webview workflow, how do they get it in without the CLI? This is the thing we haven't solved yet. By the way, just adding plugin source does not solve the problem totally, they have to add entries in config.xml, etc. It is not a trivial problem.